src/api/cvc4cpp.cpp

```diff
@@ -2124,6 +2141,61 @@ Term Solver::mkBitVector(const std::string& s, uint32_t base) const
   return mkBVFromStrHelper(s, base);
 }
 
+Term Solver::mkBitVector(uint32_t size, const char* s, uint32_t base) const
+{
+  std::string str(s);
+  return mkBitVector(size, str, base);
```

Comment:
Do not use nested API calls, will lead to problems with API tracing.
Introduce a new variant of `mkBVFromStrHelper()` and use this variant here and below.

Reply:
Good point. Done.